UE/Application/Models/Sms.cpp

```diff
@@ -0,0 +1,13 @@
+#include "Sms.hpp"
+
+namespace ue
+{
+    // Implementacja konstruktorow
```

Comment:
[ENG] It seems to me that we should only use English. After all, we are doing this as a professional. :100: